From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: violetc <58360096+s-yh-china@users.noreply.github.com>
Date: Mon, 3 Feb 2025 16:51:01 +0800
Subject: [PATCH] Leaves: Syncmatica Protocol

Original license: GPL-3.0-only
Original project: https://github.com/LeavesMC/Leaves

This patch is Powered by Syncmatica (https://github.com/End-Tech/syncmatica)

diff --git a/net/minecraft/server/network/ServerGamePacketListenerImpl.java b/net/minecraft/server/network/ServerGamePacketListenerImpl.java
index bd050d17e41f24dc4c8df57182b9c466d7276c32..c660b92547892d4ab505466d65fb88f7ab0898e4 100644
--- a/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+++ b/net/minecraft/server/network/ServerGamePacketListenerImpl.java
@@ -343,8 +343,11 @@ public class ServerGamePacketListenerImpl
         this.chatMessageChain = new FutureChain(server.chatExecutor); // CraftBukkit - async chat
         this.tickEndEvent = new io.papermc.paper.event.packet.ClientTickEndEvent(player.getBukkitEntity()); // Paper - add client tick end event
         this.playerGameConnection = new io.papermc.paper.connection.PaperPlayerGameConnection(this); // Paper
+        this.exchangeTarget = new org.leavesmc.leaves.protocol.syncmatica.exchange.ExchangeTarget(this); // Leaves - Syncmatica Protocol
     }
 
+    public final org.leavesmc.leaves.protocol.syncmatica.exchange.ExchangeTarget exchangeTarget; // Leaves - Syncmatica Protocol
+
     // Paper start - configuration phase API
     @Override
     public io.papermc.paper.connection.PlayerCommonConnection getApiConnection() {
